astroquery/nist/core.py

```diff
@@ -85,7 +85,13 @@ def _args_to_payload(self, *args, **kwargs):
 
         """
         request_payload = {}
-        request_payload["spectra"] = kwargs['linename']
+        linename = kwargs["linename"]
+        if isinstance(linename, str):
+            request_payload["spectra"] = linename
+        elif isinstance(linename, list):
+            request_payload["spectra"] = " ".join(linename)
```

Comment:
Here it just supports str and list types to produce a single str if a list is passed in "H I Fe I", so if `linename = "H I Fe I"` then its already in the right format for the request payload.